docs/pages/code/e.php

```diff
@@ -21,7 +21,7 @@
 
 $fact = static function (int $number) use ($multiplication): float {
     return Collection::range(1, $number + 1)
-        ->foldleft(
+        ->foldLeft(
```

Comment:
these are showing as PHPStan errors when running GrumPHP on local, however they don't cause it to fail. Why is that? There are others in the source code too:

```
 ------ ------------------------------------------------------------
  Line   src/Operation/Permutate.php
 ------ ------------------------------------------------------------
  57     Strict comparison using === between array() and array<int,
         T>&nonEmpty will always evaluate to false.
 ------ ------------------------------------------------------------

 ------ ---------------------------------------------------------------------
  Line   src/Operation/Times.php
 ------ ---------------------------------------------------------------------
  52     Anonymous function should return EmptyIterator but return statement
         is missing.
 ------ ---------------------------------------------------------------------

 ------ -------------------------------------------------------------
  Line   src/Operation/Unwindow.php
 ------ -------------------------------------------------------------
  35     Anonymous function should have native return typehint "?T".
  39     Foreach overwrites $value with its value variable.
 ------ -------------------------------------------------------------

Reply:
Indeed, the following Grumphp tasks:
- composer_require_checker
- phpstan
- psalm

Are set to not block the checks in case of failure.

This is

Reply:
This is on my TODO list, but feel free to implement it, I'm curious to see how it's done!

Reply:
sure, I've worked with phpstan and psalm baselines before. I'm just not sure how the whole setup works using GrumPHP, but I can look into it